src/controller/CHIPDeviceController.cpp

```diff
@@ -1631,6 +1649,57 @@ void AttributeReadFailure(void * context, CHIP_ERROR status)
     commissioner->CommissioningStageComplete(status);
 }
 
+void DeviceCommissioner::OnDone()
```

Comment:
Worth documenting that this is the ReadClient::Callback::OnDone.

Reply:
done.